src/classes/BGE_ConfigurationWizard_CTRL.cls

```diff
@@ -455,71 +438,40 @@ public with sharing class BGE_ConfigurationWizard_CTRL {
     */
     public static Map<String, String> getLabels(String sObjectName) {
```

Comment:
I'd suggest removing the fork to grab getBatchLabels() and just get all the labels here.